File: client/Client/Network/Messages/Session/CreateSessionMessage.cs
using LoadRunnerClient.DTOs;

namespace LoadRunnerClient.Messages
{
    public class CreateSessionMessage
    {
        public const string TYPE = "CreateSessionMessage";
        private SessionDTO _sessionDTO;
        public SessionDTO sessionDTO { get => _sessionDTO; set => _sessionDTO = value; }

        public CreateSessionMessage()
        {
        }

        public CreateSessionMessage(SessionDTO sessionDTO)
        {
            this.sessionDTO = sessionDTO;
        }
    }
}

File: server/src/main/java/de/hsrm/mi/swtpro03/server/network/messages/session/CreateSessionMessage.java
package de.hsrm.mi.swtpro03.server.network.messages.session;

import de.hsrm.mi.swtpro03.server.game.dtos.SessionDTO;

public class CreateSessionMessage {

	public static final String TYPE = "CreateSessionMessage";
	private String sessionName;
	private SessionDTO sessionDTO;

	public CreateSessionMessage() {
	}

	public CreateSessionMessage(SessionDTO sessionDTO) {
		this.sessionDTO = sessionDTO;
	}

	public SessionDTO getSessionDTO() {
		return sessionDTO;
	}

	public void setSessionDTO(SessionDTO sessionDTO) {
		this.sessionDTO = sessionDTO;
	}
}
